src/pages/app.react.jsx

```diff
@@ -36,6 +37,11 @@ export default React.createClass({
       mainDiv: {
         color: '#565a5f',
         backgroundColor: '#f5f8f9'
+      },
+      MainContainer: {
```

Comment:
in most of cases keys should be in `camelCase` but I don't know what kind of convention we have here :)